curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b599621f856059e2e7501.md

```diff
@@ -0,0 +1,107 @@
+---
+id: 646b599621f856059e2e7501
+title: Step 21
+challengeType: 0
+dashedName: step-21
+---
+
+# --description--
+
+Create a `p` HTML element with the `id` attribute set to `current-date`
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed